modules/desktop/graphics/hardware.nix

```
@@ -0,0 +1,14 @@
+# Copyright 2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{lib, ...}:
+with lib; {
+  options.ghaf.graphics.hardware = {
+    networkDevice = mkOption {
```

To me it looks like this is plural:

```suggestion
    networkDevices = mkOption {
```